tests/nn/test_inference.py

```diff
@@ -1381,3 +1386,42 @@ def test_flow_tracker(centered_pair_predictions: Labels, tmpdir):
         for key in tracker.candidate_maker.shifted_instances.keys():
             assert lf.frame_idx - key[0] <= track_window  # Keys are pruned
             assert abs(key[0] - key[1]) <= track_window  # References within window
+
+
+def test_movenet_inference():
+    inference_layer = MoveNetInferenceLayer(model_name="lightning")
+    inference_model = MoveNetInferenceModel(inference_layer)
+
+    video = sleap.Video.from_image_filenames(["tests/data/videos/white_guy.png"])
+
+    p = sleap.pipelines.Pipeline(sleap.pipelines.VideoReader(video))
+    p += sleap.pipelines.SizeMatcher(
+        points_key=None,
+        max_image_width=inference_model.image_size,
+        max_image_height=inference_model.image_size,
+        center_pad=True,
+    )
+    p += sleap.pipelines.Batcher(batch_size=4)
+
+    ex = p.peek(1)
+    preds = inference_model.predict_on_batch(ex)
+    assert preds["instance_peaks"].shape == (1, 1, 17, 2)
+
+
+def test_movenet_predictor(min_dance_labels):
+    predictor = MoveNetPredictor.from_trained_models("thunder")
+    predictor.verbosity = "none"
+    assert predictor.is_grayscale == False
+    labels_pr = predictor.predict(min_dance_labels)
```

Comment:
Can we predict on less frames. We can do this by creating our own `VideoReader` instance with only a few `example_indices`:
```suggestion
    # TODO (Jiaying): Add a fixture named `movenet_video` that returns a `Video` object created from dance.mp4
    vr = VideoReader(video=movenet_video, example_indices=[0, 1, 2])
    labels_pr = predictor.predict(data=vr)
```